examples/CamelSalesforceSourceConnector.properties

```diff
@@ -0,0 +1,35 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name=CamelSalesforceSourceConnector
+connector.class=org.apache.camel.kafkaconnector.salesforce.CamelSalesforceSourceConnector
+key.converter=org.apache.kafka.connect.storage.StringConverter
+value.converter=org.apache.kafka.connect.storage.StringConverter
+
+camel.source.kafka.topic=mytopic
+
+camel.source.url=salesforce:CamelKafkaConnectorTestTopic?
```

Comment:
If you choose to use camel.source.url, you need to specify everything there and avoid camel.source.endpoint.* properties. If you choose to use camel.source.endpoint.* you must avoid camel.source.url